src/components/BusinessCard/BusinessCard.jsx

```diff
@@ -1,26 +1,60 @@
 import { useState } from "react";
 import Button from "../Button/Button";
 import DetailsPane from "../DetailsPane/DetailsPane";
+import "./BusinessCard.scss";
+import cardCategories from "../../data/favourites/cardCategoryData/cardCategories";
 
 const BusinessCard = ({ businessCardArray }) => {
   const [activeDetails, setActiveDetails] = useState(null);
-  const showCards = businessCardArray.map((card) => {
-    return (
-      <div key={card._id}>
-        <h2>{card.name}</h2>
-        <p>{card.description}</p>
-        <p>{card.address}</p>
-        <div>
-          <Button
-            buttonName="details"
-            buttonText="Details"
-            onClick={() => setActiveDetails(card)}
+
+  const getDescription = (card) => {
+    const lastSentenceIndex = card.description.indexOf(".", 2) + 1;
+    const shortenedDescription = card.description.substring(
+      0,
+      lastSentenceIndex
+    );
+
+    return shortenedDescription;
+  };
+
+  const showCards = businessCardArray.map((card) => (
+    <div className="card" key={card._id}>
+      <div className="card__header-container">
+        <h2
+          className={`card__heading card__heading--${
+            cardCategories[card.category].categoryName
```

Comment:
Might want to optional chain this in case we get something that doesn't work: `cardCategories[card.category]?.categoryName`

Reply:
cardCategories[card.category].categoryName ??
              cardCategories["Other"].categoryName